### PR TITLE
duck 4.7.3

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
-  url "https://dist.duck.sh/duck-src-4.7.2.18007.tar.gz"
-  sha256 "2b30071ef58aed42805724f318b1e3f013749cc534e54265ee305dfc2ec86d69"
+  url "https://dist.duck.sh/duck-src-4.7.3.18396.tar.gz"
+  sha256 "47e25f0a28393c388f37d319c9d51dd51eebdf15198ee48df3995af8a60bcc16"
   head "https://svn.cyberduck.io/trunk/"
-
+
```
Comment (bfontaine) [inline Library/Formula/duck.rb]: There are trailing spaces here.

**Comment by DomT4** [comment]
You may need to rebase/squash this. The CI is having trouble merging it for testing.

**Comment by mikemcquaid** [comment]
@dkocher It would be good if you manually submitted PRs

**Comment by mikemcquaid** [comment]
Or at least made it such that it's automated so that it passes consistently on submission.

**Comment by dkocher** [comment]
Will do.

**Comment by dkocher** [comment]
Rebased the branch with upstream.

**Comment by mikemcquaid** [comment]
It's failing to install: https://travis-ci.org/Homebrew/homebrew/jobs/87006817

**Comment by dkocher** [comment]
Can we update the build requirement to Java 1.8+?

**Comment by mikemcquaid** [comment]
Yep, that's fine.

**Comment by dkocher** [comment]
Changeset 24ab963e23e560bd10fa36f1686416e1e430e924 broke the test.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f6ff9c4c4d1d3f5fd4b2a3ed20f43bf82298d08b. Thank you for your contribution to Homebrew @dkocher / your Jenkins bot :wink:. Appreciated.

**Pull request closed without merging**